packages/netlify-cms-widget-select/src/SelectControl.js

```diff
@@ -79,7 +79,7 @@ export default class SelectControl extends React.Component {
   handleChange = selectedOption => {
     const { onChange, field } = this.props;
     const isMultiple = field.get('multiple', false);
-    const isEmpty = isMultiple ? !selectedOption?.length : !selectedOption;
+    const isEmpty = selectedOption === null;
```

Comment:
See https://github.com/JedWatson/react-select/releases/tag/react-select%403.0.3 under `Normalized Values`.

Clearing a multiple select will now return `null` instead of an empty array